lisa/operating_system.py

```diff
@@ -57,6 +57,9 @@ class OperatingSystem:
         r"^NAME=\"?([^\" \r\n]+)[^\" \n]*\"?\r?$", re.M
     )
     __os_release_pattern_id = re.compile(r"^ID=\"?([^\" \r\n]+)[^\" \n]*\"?\r?$", re.M)
+    __os_release_pattern_idlike = re.compile(
```

Comment:
Out of curiosity, what are values for set for ID and ID_LIKE in almalinux?

Reply:
```
NAME="AlmaLinux"
VERSION="8.4 (Electric Cheetah)"
ID="almalinux"
ID_LIKE="rhel centos fedora"
```

Reply:
Add an example above.

Reply:
Have added an example as a comment above the two line.